<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />

<meta name="viewport" content="width=device-width, initial-scale=1" />

<meta name="author" content="Hannah L. Owens" />
<meta name="author" content="Cory Merow" />
<meta name="author" content="Brian Maitner" />
<meta name="author" content="Jamie M. Kass" />
<meta name="author" content="Vijay Barve" />
<meta name="author" content="Robert Guralnick" />

<meta name="date" content="2024-10-28" />

<title>Advanced search and citation of occurrences</title>

<script>// Pandoc 2.9 adds attributes on both header and div. We remove the former (to
// be compatible with the behavior of Pandoc < 2.8).
document.addEventListener('DOMContentLoaded', function(e) {
  var hs = document.querySelectorAll("div.section[class*='level'] > :first-child");
  var i, h, a;
  for (i = 0; i < hs.length; i++) {
    h = hs[i];
    if (!/^h[1-6]$/i.test(h.tagName)) continue;  // it should be a header h1-h6
    a = h.attributes;
    while (a.length > 0) h.removeAttribute(a[0].name);
  }
});
</script>

<style type="text/css">
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
</style>



<style type="text/css">
code {
white-space: pre;
}
.sourceCode {
overflow: visible;
}
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
{ counter-reset: source-line 0; }
pre.numberSource code > span
{ position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
{ content: counter(source-line);
position: relative; left: -1em; text-align: right; vertical-align: baseline;
border: none; display: inline-block;
-webkit-touch-callout: none; -webkit-user-select: none;
-khtml-user-select: none; -moz-user-select: none;
-ms-user-select: none; user-select: none;
padding: 0 4px; width: 4em;
color: #aaaaaa;
}
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa; padding-left: 4px; }
div.sourceCode
{ }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } 
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } 
code span.at { color: #7d9029; } 
code span.bn { color: #40a070; } 
code span.bu { color: #008000; } 
code span.cf { color: #007020; font-weight: bold; } 
code span.ch { color: #4070a0; } 
code span.cn { color: #880000; } 
code span.co { color: #60a0b0; font-style: italic; } 
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } 
code span.do { color: #ba2121; font-style: italic; } 
code span.dt { color: #902000; } 
code span.dv { color: #40a070; } 
code span.er { color: #ff0000; font-weight: bold; } 
code span.ex { } 
code span.fl { color: #40a070; } 
code span.fu { color: #06287e; } 
code span.im { color: #008000; font-weight: bold; } 
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } 
code span.kw { color: #007020; font-weight: bold; } 
code span.op { color: #666666; } 
code span.ot { color: #007020; } 
code span.pp { color: #bc7a00; } 
code span.sc { color: #4070a0; } 
code span.ss { color: #bb6688; } 
code span.st { color: #4070a0; } 
code span.va { color: #19177c; } 
code span.vs { color: #4070a0; } 
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } 
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>




<style type="text/css">body {
background-color: #fff;
margin: 1em auto;
max-width: 700px;
overflow: visible;
padding-left: 2em;
padding-right: 2em;
font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
font-size: 14px;
line-height: 1.35;
}
#TOC {
clear: both;
margin: 0 0 10px 10px;
padding: 4px;
width: 400px;
border: 1px solid #CCCCCC;
border-radius: 5px;
background-color: #f6f6f6;
font-size: 13px;
line-height: 1.3;
}
#TOC .toctitle {
font-weight: bold;
font-size: 15px;
margin-left: 5px;
}
#TOC ul {
padding-left: 40px;
margin-left: -1.5em;
margin-top: 5px;
margin-bottom: 5px;
}
#TOC ul ul {
margin-left: -2em;
}
#TOC li {
line-height: 16px;
}
table {
margin: 1em auto;
border-width: 1px;
border-color: #DDDDDD;
border-style: outset;
border-collapse: collapse;
}
table th {
border-width: 2px;
padding: 5px;
border-style: inset;
}
table td {
border-width: 1px;
border-style: inset;
line-height: 18px;
padding: 5px 5px;
}
table, table th, table td {
border-left-style: none;
border-right-style: none;
}
table thead, table tr.even {
background-color: #f7f7f7;
}
p {
margin: 0.5em 0;
}
blockquote {
background-color: #f6f6f6;
padding: 0.25em 0.75em;
}
hr {
border-style: solid;
border: none;
border-top: 1px solid #777;
margin: 28px 0;
}
dl {
margin-left: 0;
}
dl dd {
margin-bottom: 13px;
margin-left: 13px;
}
dl dt {
font-weight: bold;
}
ul {
margin-top: 0;
}
ul li {
list-style: circle outside;
}
ul ul {
margin-bottom: 0;
}
pre, code {
background-color: #f7f7f7;
border-radius: 3px;
color: #333;
white-space: pre-wrap; 
}
pre {
border-radius: 3px;
margin: 5px 0px 10px 0px;
padding: 10px;
}
pre:not([class]) {
background-color: #f7f7f7;
}
code {
font-family: Consolas, Monaco, 'Courier New', monospace;
font-size: 85%;
}
p > code, li > code {
padding: 2px 0px;
}
div.figure {
text-align: center;
}
img {
background-color: #FFFFFF;
padding: 2px;
border: 1px solid #DDDDDD;
border-radius: 3px;
border: 1px solid #CCCCCC;
margin: 0 5px;
}
h1 {
margin-top: 0;
font-size: 35px;
line-height: 40px;
}
h2 {
border-bottom: 4px solid #f7f7f7;
padding-top: 10px;
padding-bottom: 2px;
font-size: 145%;
}
h3 {
border-bottom: 2px solid #f7f7f7;
padding-top: 10px;
font-size: 120%;
}
h4 {
border-bottom: 1px solid #f7f7f7;
margin-left: 8px;
font-size: 105%;
}
h5, h6 {
border-bottom: 1px solid #ccc;
font-size: 105%;
}
a {
color: #0033dd;
text-decoration: none;
}
a:hover {
color: #6666ff; }
a:visited {
color: #800080; }
a:visited:hover {
color: #BB00BB; }
a[href^="http:"] {
text-decoration: underline; }
a[href^="https:"] {
text-decoration: underline; }

code > span.kw { color: #555; font-weight: bold; } 
code > span.dt { color: #902000; } 
code > span.dv { color: #40a070; } 
code > span.bn { color: #d14; } 
code > span.fl { color: #d14; } 
code > span.ch { color: #d14; } 
code > span.st { color: #d14; } 
code > span.co { color: #888888; font-style: italic; } 
code > span.ot { color: #007020; } 
code > span.al { color: #ff0000; font-weight: bold; } 
code > span.fu { color: #900; font-weight: bold; } 
code > span.er { color: #a61717; background-color: #e3d2d2; } 
</style>




</head>

<body>




<h1 class="title toc-ignore">Advanced search and citation of
occurrences</h1>
<h4 class="author">Hannah L. Owens</h4>
<h4 class="author">Cory Merow</h4>
<h4 class="author">Brian Maitner</h4>
<h4 class="author">Jamie M. Kass</h4>
<h4 class="author">Vijay Barve</h4>
<h4 class="author">Robert Guralnick</h4>
<h4 class="date">2024-10-28</h4>


<div id="TOC">
<ul>
<li><a href="#advanced-features" id="toc-advanced-features">Advanced
features</a>
<ul>
<li><a href="#loading-data-from-previous-gbif-searches" id="toc-loading-data-from-previous-gbif-searches">Loading data from
previous GBIF searches</a></li>
<li><a href="#performing-a-multi-species-search" id="toc-performing-a-multi-species-search">Performing a Multi-Species
Search</a></li>
<li><a href="#occcite-with-a-phylogeny" id="toc-occcite-with-a-phylogeny">occCite with a Phylogeny</a></li>
</ul></li>
</ul>
</div>

<div id="advanced-features" class="section level1">
<h1>Advanced features</h1>
<p>This vignette demonstrates more advanced features and customization
available in <code>occCite</code>. We recommend you read
<code>vignette(&quot;Simple.Rmd&quot;, package = &quot;occCite&quot;)</code> first, if you
have not already done so.</p>
<div id="loading-data-from-previous-gbif-searches" class="section level2">
<h2>Loading data from previous GBIF searches</h2>
<p>Querying GBIF can take quite a bit of time, especially for multiple
species and/or well-known species. In this case, you may wish to access
previously-downloaded data sets from your computer by specifying the
general location of your downloaded <code>.zip</code> files.
<code>occQuery</code> will crawl through your specified
<code>GBIFDownloadDirectory</code> to collect all the <code>.zip</code>
files contained in that folder and its subfolders. It will then import
the most recent downloads that match your taxon list. These GBIF data
will be appended to a BIEN search the same as if you do the simple
real-time search (if you chose BIEN as well as GBIF), as was shown
above. <code>checkPreviousGBIFDownload</code> is <code>TRUE</code> by
default, but if <code>loadLocalGBIFDownload</code> is <code>TRUE</code>,
<code>occQuery</code> will ignore <code>checkPreviousDownload</code>. It
is also worth noting that <code>occCite</code> does not currently
support mixed data download sources. That is, you cannot do GBIF queries
for some taxa, download previously-prepared data sets for others, and
load the rest from local data sets on your computer.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" tabindex="-1"></a><span class="co"># Simple search</span></span>
<span id="cb1-2"><a href="#cb1-2" tabindex="-1"></a>myOldOccCiteObject <span class="ot">&lt;-</span> <span class="fu">occQuery</span>(<span class="at">x =</span> <span class="st">&quot;Protea cynaroides&quot;</span>,</span>
<span id="cb1-3"><a href="#cb1-3" tabindex="-1"></a>                                  <span class="at">datasources =</span> <span class="fu">c</span>(<span class="st">&quot;gbif&quot;</span>, <span class="st">&quot;bien&quot;</span>),</span>
<span id="cb1-4"><a href="#cb1-4" tabindex="-1"></a>                                  <span class="at">GBIFLogin =</span> GBIFLogin, </span>
<span id="cb1-5"><a href="#cb1-5" tabindex="-1"></a>                                  <span class="at">GBIFDownloadDirectory =</span> </span>
<span id="cb1-6"><a href="#cb1-6" tabindex="-1"></a>                                    <span class="fu">system.file</span>(<span class="st">&#39;extdata/&#39;</span>, <span class="at">package=</span><span class="st">&#39;occCite&#39;</span>),</span>
<span id="cb1-7"><a href="#cb1-7" tabindex="-1"></a>                                  <span class="at">checkPreviousGBIFDownload =</span> T)</span></code></pre></div>
<p>Here is the result. Look familiar?</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" tabindex="-1"></a><span class="co">#GBIF search results</span></span>
<span id="cb2-2"><a href="#cb2-2" tabindex="-1"></a><span class="fu">head</span>(myOldOccCiteObject<span class="sc">@</span>occResults<span class="sc">$</span><span class="st">`</span><span class="at">Protea cynaroides</span><span class="st">`</span><span class="sc">$</span>GBIF<span class="sc">$</span>OccurrenceTable);</span></code></pre></div>
<pre><code>##                name longitude  latitude coordinateUncertaintyInMeters day month
## 1 Protea cynaroides  18.43928 -33.95440                             8  17     2
## 2 Protea cynaroides  22.12754 -33.91561                             4  11     2
## 3 Protea cynaroides  18.43927 -33.95429                             8  17     2
## 4 Protea cynaroides  18.43254 -34.29275                            31   6     2
## 5 Protea cynaroides  18.42429 -34.02934                          2167  10     2
## 6 Protea cynaroides  18.43529 -34.10545                             2   8     2
##   year                           datasetKey dataService
## 1 2022 50c9509d-22c7-4a22-a47d-8c48425ef4a7        GBIF
## 2 2022 50c9509d-22c7-4a22-a47d-8c48425ef4a7        GBIF
## 3 2022 50c9509d-22c7-4a22-a47d-8c48425ef4a7        GBIF
## 4 2022 50c9509d-22c7-4a22-a47d-8c48425ef4a7        GBIF
## 5 2022 50c9509d-22c7-4a22-a47d-8c48425ef4a7        GBIF
## 6 2022 50c9509d-22c7-4a22-a47d-8c48425ef4a7        GBIF
##                               datasetName
## 1 iNaturalist Research-grade Observations
## 2 iNaturalist Research-grade Observations
## 3 iNaturalist Research-grade Observations
## 4 iNaturalist Research-grade Observations
## 5 iNaturalist Research-grade Observations
## 6 iNaturalist Research-grade Observations</code></pre>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" tabindex="-1"></a><span class="co">#The full summary</span></span>
<span id="cb4-2"><a href="#cb4-2" tabindex="-1"></a><span class="fu">summary</span>(myOldOccCiteObject)</span></code></pre></div>
<pre><code>##  
##  OccCite query occurred on: 20 June, 2024
##  
##  User query type: User-supplied list of taxa.
##  
##  Sources for taxonomic rectification: GBIF Backbone Taxonomy
##      
##  
##  Taxonomic cleaning results:     
## 
##          Input Name                Best Match Taxonomic Databases w/ Matches
## 1 Protea cynaroides Protea cynaroides (L.) L.         GBIF Backbone Taxonomy
##  
##  Sources for occurrence data: gbif, bien
##      
##                     Species Occurrences Sources
## 1 Protea cynaroides (L.) L.        2334      17
##  
##  GBIF dataset DOIs:  
## 
##                     Species GBIF Access Date           GBIF DOI
## 1 Protea cynaroides (L.) L.       2022-03-02 10.15468/dl.ztbx8c</code></pre>
<p>Getting citation data works the exact same way with
previously-downloaded data as it does from a fresh data set.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" tabindex="-1"></a><span class="co">#Get citations</span></span>
<span id="cb6-2"><a href="#cb6-2" tabindex="-1"></a>myOldOccCitations <span class="ot">&lt;-</span> <span class="fu">occCitation</span>(myOldOccCiteObject)</span>
<span id="cb6-3"><a href="#cb6-3" tabindex="-1"></a><span class="fu">print</span>(myOldOccCitations)</span></code></pre></div>
<pre><code>## Writing 5 Bibtex entries ... OK
## Results written to file &#39;temp.bib&#39;</code></pre>
<pre><code>## AFFOUARD A, JOLY A, LOMBARDO J, CHAMP J, GOEAU H, CHOUET M, GRESSE H, BONNET P (2023). Pl@ntNet observations. Version 1.8. Pl@ntNet. https://doi.org/10.15468/gtebaa. Accessed via GBIF on 2022-03-02.
## AFFOUARD A, JOLY A, LOMBARDO J, CHAMP J, GOEAU H, CHOUET M, GRESSE H, BOTELLA C, BONNET P (2023). Pl@ntNet automatically identified occurrences. Version 1.8. Pl@ntNet. https://doi.org/10.15468/mma2ec. Accessed via GBIF on 2022-03-02.
## Chamberlain, S., Barve, V., Mcglinn, D., Oldoni, D., Desmet, P., Geffert, L., Ram, K. (2024). rgbif: Interface to the Global Biodiversity Information Facility API. R package version 3.8.1. https://CRAN.R-project.org/package = rgbif.
## Chamberlain, S., Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. PeerJ PrePrints.
## Fatima Parker-Allie, Ranwashe F (2018). PRECIS. South African National Biodiversity Institute. https://doi.org/10.15468/rckmn2. Accessed via GBIF on 2022-03-02.
## MNHN, Chagnoux S (2024). The vascular plants collection (P) at the Herbarium of the Muséum national d&#39;Histoire Naturelle (MNHN - Paris). Version 69.384. MNHN - Museum national d&#39;Histoire naturelle. https://doi.org/10.15468/nc6rxy. Accessed via GBIF on 2022-03-02.
## MNHN. Accessed via BIEN on NA.
## Maitner, B. (2023). . R package version 1.2.6. https://CRAN.R-project.org/package = BIEN.
## Missouri Botanical Garden,Herbarium. Accessed via BIEN on NA.
## Observation.org (2024). Observation.org, Nature data from around the World. https://doi.org/10.15468/5nilie. Accessed via GBIF on 2022-03-02.
## Owens, H., Merow, C., Maitner, B., Kass, J., Barve, V., Guralnick, R. (2024). occCite: Querying and Managing Large Biodiversity Occurrence Datasets. R package version 0.5.9. https://CRAN.R-project.org/package = occCite.
## Ranwashe F (2024). Botanical Database of Southern Africa (BODATSA): Botanical Collections. Version 1.25. South African National Biodiversity Institute. https://doi.org/10.15468/2aki0q. Accessed via GBIF on 2022-03-02.
## Rob Cubey (2022). Royal Botanic Garden Edinburgh Living Plant Collections (E). Royal Botanic Garden Edinburgh. https://doi.org/10.15468/bkzv1l. Accessed via GBIF on 2022-03-02.
## SANBI. Accessed via BIEN on NA.
## Senckenberg (2020). African Plants - a photo guide. https://doi.org/10.15468/r9azth. Accessed via GBIF on 2022-03-02.
## Taylor S (2019). G. S. Torrey Herbarium at the University of Connecticut (CONN). University of Connecticut. https://doi.org/10.15468/w35jmd. Accessed via GBIF on 2022-03-02.
## Team}, {.C. (2024). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/.
## Teisher J, Stimmel H (2024). Tropicos MO Specimen Data. Missouri Botanical Garden. https://doi.org/10.15468/hja69f. Accessed via GBIF on 2022-03-02.
## Tela Botanica. Carnet en Ligne. https://doi.org/10.15468/rydcn2. Accessed via GBIF on 2022-03-02.
## UConn. Accessed via BIEN on NA.
## iNaturalist contributors, iNaturalist (2024). iNaturalist Research-grade Observations. iNaturalist.org. https://doi.org/10.15468/ab3s5x. Accessed via GBIF on 2022-03-02.
## naturgucker.de. naturgucker. https://doi.org/10.15468/uc1apo. Accessed via GBIF on 2022-03-02.</code></pre>
<p>Note that you can also load multiple species using either a vector of
species names or a phylogeny (provided you have previously downloaded
data for all of the species of interest), and you can load occurrences
from non-GBIF data sources (e.g. BIEN) in the same query.</p>
<hr />
</div>
<div id="performing-a-multi-species-search" class="section level2">
<h2>Performing a Multi-Species Search</h2>
<p>In addition to doing a simple, single species search, you can also
use <code>occCite</code> to search for and manage occurrence datasets
for multiple species. You can either submit a vector of species names,
or you can submit a <em>phylogeny</em>! The occCitation function will
return a named list of citation tables in the case of multiple
species.</p>
</div>
<div id="occcite-with-a-phylogeny" class="section level2">
<h2>occCite with a Phylogeny</h2>
<p>Here is an example of how such a search is structured, using an
unpublished phylogeny of billfishes.</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" tabindex="-1"></a><span class="fu">library</span>(ape)</span>
<span id="cb9-2"><a href="#cb9-2" tabindex="-1"></a><span class="co">#Get tree</span></span>
<span id="cb9-3"><a href="#cb9-3" tabindex="-1"></a>treeFile <span class="ot">&lt;-</span> <span class="fu">system.file</span>(<span class="st">&quot;extdata/Fish_12Tax_time_calibrated.tre&quot;</span>, <span class="at">package=</span><span class="st">&#39;occCite&#39;</span>)</span>
<span id="cb9-4"><a href="#cb9-4" tabindex="-1"></a>phylogeny <span class="ot">&lt;-</span> ape<span class="sc">::</span><span class="fu">read.nexus</span>(treeFile)</span>
<span id="cb9-5"><a href="#cb9-5" tabindex="-1"></a>tree <span class="ot">&lt;-</span> ape<span class="sc">::</span><span class="fu">extract.clade</span>(phylogeny, <span class="dv">22</span>)</span>
<span id="cb9-6"><a href="#cb9-6" tabindex="-1"></a><span class="co">#Query databases for names</span></span>
<span id="cb9-7"><a href="#cb9-7" tabindex="-1"></a>myPhyOccCiteObject <span class="ot">&lt;-</span> <span class="fu">studyTaxonList</span>(<span class="at">x =</span> tree, </span>
<span id="cb9-8"><a href="#cb9-8" tabindex="-1"></a>                                     <span class="at">datasources =</span> <span class="st">&quot;GBIF Backbone Taxonomy&quot;</span>)</span></code></pre></div>
<pre><code>## handled warning: Package taxize unavailable. Skipping taxonomic rectification.
## handled warning: Package taxize unavailable. Skipping taxonomic rectification.
## handled warning: Package taxize unavailable. Skipping taxonomic rectification.</code></pre>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" tabindex="-1"></a><span class="co">#Query GBIF for occurrence data</span></span>
<span id="cb11-2"><a href="#cb11-2" tabindex="-1"></a>myPhyOccCiteObject <span class="ot">&lt;-</span> <span class="fu">occQuery</span>(<span class="at">x =</span> myPhyOccCiteObject, </span>
<span id="cb11-3"><a href="#cb11-3" tabindex="-1"></a>                            <span class="at">datasources =</span> <span class="st">&quot;gbif&quot;</span>,</span>
<span id="cb11-4"><a href="#cb11-4" tabindex="-1"></a>                            <span class="at">GBIFDownloadDirectory =</span> <span class="fu">system.file</span>(<span class="st">&#39;extdata/&#39;</span>, <span class="at">package=</span><span class="st">&#39;occCite&#39;</span>),</span>
<span id="cb11-5"><a href="#cb11-5" tabindex="-1"></a>                            <span class="at">loadLocalGBIFDownload =</span> T,</span>
<span id="cb11-6"><a href="#cb11-6" tabindex="-1"></a>                            <span class="at">checkPreviousGBIFDownload =</span> F)</span></code></pre></div>
<pre><code>## Warning in gbifRetriever(searchTaxa[[i]]): GBIF unreachable; please try again later.
## Warning in gbifRetriever(searchTaxa[[i]]): GBIF unreachable; please try again later.
## Warning in gbifRetriever(searchTaxa[[i]]): GBIF unreachable; please try again later.</code></pre>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" tabindex="-1"></a><span class="co"># What does a multispecies query look like?</span></span>
<span id="cb13-2"><a href="#cb13-2" tabindex="-1"></a><span class="fu">summary</span>(myPhyOccCiteObject)</span></code></pre></div>
<pre><code>##  
##  OccCite query occurred on: 28 October, 2024
##  
##  User query type: User-supplied phylogeny.
##  
##  Sources for taxonomic rectification: GBIF Backbone Taxonomy
##      
##  
##  Taxonomic cleaning results:     
## 
##                   Input Name                 Best Match
## 1 Tetrapturus_angustirostris Tetrapturus_angustirostris
## 2         Tetrapturus_belone         Tetrapturus_belone
## 3      Tetrapturus_pfluegeri      Tetrapturus_pfluegeri
##   Taxonomic Databases w/ Matches
## 1                 Not rectified.
## 2                 Not rectified.
## 3                 Not rectified.
##  
##  Sources for occurrence data: gbif
##      
##                      Species Occurrences Sources
## 1 Tetrapturus_angustirostris           0       0
## 2         Tetrapturus_belone           0       0
## 3      Tetrapturus_pfluegeri           0       0
##  
##  GBIF dataset DOIs:  
## 
##                      Species GBIF Access Date GBIF DOI
## 1 Tetrapturus_angustirostris             &lt;NA&gt;     &lt;NA&gt;
## 2         Tetrapturus_belone             &lt;NA&gt;     &lt;NA&gt;
## 3      Tetrapturus_pfluegeri             &lt;NA&gt;     &lt;NA&gt;</code></pre>
<p>When you have results for multiple species, as in this case, you can
also plot the summary figures either for the whole search…</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" tabindex="-1"></a><span class="fu">plot</span>(myPhyOccCiteObject)</span></code></pre></div>
<pre><code>## Error in d.tbl[[i]]: subscript out of bounds</code></pre>
<p><em>or</em> you can plot the results by species!</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" tabindex="-1"></a><span class="fu">plot</span>(myPhyOccCiteObject, <span class="at">bySpecies =</span> T, <span class="at">plotTypes =</span> <span class="fu">c</span>(<span class="st">&quot;yearHistogram&quot;</span>, <span class="st">&quot;source&quot;</span>))</span></code></pre></div>
<pre><code>## Error in d.tbl[[i]]: subscript out of bounds</code></pre>
<p>And then you can print out the citations, separated by species (or
not, but in this example, they’re separate).</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" tabindex="-1"></a><span class="co">#Get citations</span></span>
<span id="cb19-2"><a href="#cb19-2" tabindex="-1"></a>myPhyOccCitations <span class="ot">&lt;-</span> <span class="fu">occCitation</span>(myPhyOccCiteObject)</span></code></pre></div>
<pre><code>## Error in strsplit(occResults$GBIF$Metadata$modified, &quot;T&quot;): non-character argument</code></pre>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" tabindex="-1"></a><span class="co">#Print citations as text with accession dates.</span></span>
<span id="cb21-2"><a href="#cb21-2" tabindex="-1"></a><span class="fu">print</span>(myPhyOccCitations, <span class="at">bySpecies =</span> T)</span></code></pre></div>
<pre><code>## Error: object &#39;myPhyOccCitations&#39; not found</code></pre>
</div>
</div>



<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
